docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -116,6 +116,19 @@ export GCS_SECRET='wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'
 ```
 ````
 
+Streaming dataset also supports [GCP service account credentials](https://cloud.google.com/iam/docs/service-account-overview). Users must set their GCP `account credentials` in the run environment.
```

Comment:
Can you please create a two sections, one for `GCP Service Account Credentials` and another for `GCP User Auth Credentials Mounted as Environment Variables` similar to [this](https://docs.mosaicml.com/projects/mcli/en/latest/resources/secrets/gcp.html) doc. Thanks!